test_relationship_server.py

```diff
@@ -0,0 +1,146 @@
+import pytest
```

Comment:
How can we run this test without commenting `app.run`?

Reply:
Resolved as per discussion on Telegram. Deciding to use the commented `app.run` option. 

Based on @meghanabhange's information that PythonAnywhere makes us comment that anyway.